packages/api/src/controllers/v1/collector/mod.rs

```diff
@@ -0,0 +1,249 @@
+#![allow(unused)]
```

Comment:
avoid to allow waning for a file.
instead of it, allow each line if you must need it.